src/components/ContentItem.astro

```diff
@@ -0,0 +1,27 @@
+---
+interface Props {
+    name: string;
+    image: string;
+    href: string
+}
+
+const { name, image, href } = Astro.props;
+---
+
+<div class="item">
+    <a href={href}>
+        <img src={image} alt={name} />
```

Comment:
This should _probably_ be `alt=""` instead? @davidleininger whatcha think?

Reply:
In this case, it's probably decorative since the image and text are virtually the same. I agree we should do `alt=""`.